.azure-pipelines/build-job.yml

```diff
@@ -0,0 +1,203 @@
+parameters:
+
+- name: jobName
+  type: string
+
+- name: displayName
+  type: string
+
+- name: pool
+  type: object
+
+- name: container
+  type: string
+  default: ''
+
+- name: timeoutInMinutes
+  type: number
+  default: 0
```

Comment:
Rather than making this 0 and having an if below, should we just make the default 60? That way we don't need the if and we're explicitly calling out what our default is (which is the same as the system default)

Reply:
Good call.